tests/std/tests/P0896R4_ranges_alg_unique/test.cpp

```diff
@@ -45,6 +45,15 @@ struct instantiator {
             assert(comparisonCounter == size(input) - 1);
         }
 
+        { // Validate already unique output returns empty subrange
+            P input[4] = {{0, 99}, {1, 47}, {3, 99}, {4, 47}};
+            ReadWrite wrapped_input{input};
+
+            auto result = unique(wrapped_input.begin(), wrapped_input.end(), countedEq, get_second);
+            STATIC_ASSERT(same_as<decltype(result), subrange<iterator_t<ReadWrite>>>);
+            assert(result.empty());

```

Comment:
Observation, no change requested: This could be more strict, by verifying that the empty subrange is at the end. However, it's unlikely that we'll damage this (and this PR certainly isn't), so it's just something to keep in mind for future tests.